internal/db/constants.go

```diff
@@ -0,0 +1,5 @@
+package db
+
+const CollectionUsers = "users"
+const CollectionPosts = "posts"
+const FieldId = "_id"
```

Comment:
didn't run go fmt here? install golang linter in your ide if not done already

Reply:
when i save the file, this is the format it always fallback to, if you've any specific linting requirements, can you suggest a commit please?

Reply:
Okay linter makes it this way.. then its cool. closing